testgen/generators/collation_short.py

```diff
@@ -142,7 +142,7 @@ def check_parse_compare(self, line_index, lines):
                 raw_string2 = is_comparison_match.group(3)
                 string2 = ''
                 try:
-                    string2 = raw_string2.encode().decode("unicode_escape")
+                    string2 = raw_string2  # Don't do any unescaping
```

Comment:
Hmm, if we're not unescaping the escape sequences, then how are we comparing the intended characters in the string?...

Reply:
I will look at handling these in the test **driver**, making sure that we do indeed send the right characters to the executors.